dvc/objects/db/oss.py

```diff
@@ -0,0 +1,5 @@
+from .base import ObjectDB
+
+
+class OSSObjectDB(ObjectDB):
+    DEFAULT_VERIFY = True
```

Comment:
Would you mind documenting the purpose of this? From an outside look, it is not clear why we have this separate ODB for OSS, so a few inline comments might help.

Reply:
@isidentical It is the same as with gdrive. This is the flag we use to tell dvc pull to not trust md5s and actually verify them by hand after downloading. A legacy thing that is a bit clumsy, it will be cleaned up during save/transfer unification, I think.

Reply:
> This is the flag we use to tell dvc pull to not trust md5s and actually verify them by hand after downloading. 

Yeah, I know what it does but what I mean is that why do we need to turn it on for OSS?

Reply:
@isidentical Ah, gotcha. We've agreed with @karajan1001 to turn it on temporarily for now just as an extra assurance. We'll turn it off once we are ready. But yeah, a comment about it would be nice to avoid confusion :+1:

Reply:
Will add them later. This PR is still in progress. Because of some of the performance optimizations. 
I will submit the result of dvc-bench for it.